### PR TITLE
fixes: warning MSB4011 Microsoft.NET.Sdk.WindowsDesktop.props cannot …

### PR DESCRIPTION
**Pull request by workgroupengineering**
…be imported again.

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by grokys** [comment]
Can we get some explanation of what this is doing? Links to documentation, etc?

**Comment by kekekeks** [comment]
We probably don't even need MSBuild.Sdk.Extras with .NET 6 SDK

**Comment by workgroupengineering** [comment]
> Can we get some explanation of what this is doing? Links to documentation, etc?

I have no documentation, I was able to solve via the binlog.



> We probably don't even need MSBuild.Sdk.Extras with .NET 6 SDK

probably. When I created this PR the .NET SDK it was 5.0

File: samples/interop/WindowsInteropTest/WindowsInteropTest.csproj [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (maxkatz6) [inline samples/interop/WindowsInteropTest/WindowsInteropTest.csproj:1]: I don't think we need this line

**Review by maxkatz6 — DISMISSED** [review]

**Comment by grokys** [comment]
This seemed to have been force pushed to an older version and was showing up changes from unrelated PRs. I've updated the branch from master which means it's not not showing changes from other PRs, but I'm not 100% sure that the branch is in the state which @maxkatz6 approved it for.

@maxkatz6 does this still look right? I've removed your approval until you can check

**Comment by maxkatz6** [comment]
@grokys yes, it looks look now

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**